docker-compose/mariadb-config/prod/mariadb.cnf

```diff
@@ -0,0 +1,58 @@
+[mariadb]
+
+port                  = 3306
+bind-address          = 0.0.0.0
+
+# Fine tuning
+max_connections         = 300
+connect_timeout         = 5
+wait_timeout            = 600
+max_allowed_packet      = 32M
+thread_cache_size       = 128
+sort_buffer_size        = 128M
+bulk_insert_buffer_size = 128M
+tmp_table_size          = 32M
+max_heap_table_size     = 32M
+skip-name-resolve       = 1
+innodb_log_file_size    = 2G
+
+# Logging
+# warning log-basename is defined later for replication
+# some log option may be ignored.
+# see: https://mariadb.com/kb/en/mysqld-options/#-log-basename
+# log_error = /var/log/mysql/error.log
+slow_query_log
+slow_query_log_file    = /var/log/mysql/mariadb-slow.log
+long_query_time        = 2
+log_slow_verbosity     = query_plan,explain
+log_error              = /var/log/mysql/mariadb.err.log
+
+# Query cache
+query_cache_size        = 16M
+
+# Character sets
+character-set-server = utf8mb4
+collation-server     = utf8mb4_general_ci
+
+# InnoDB
+# InnoDB is enabled by default with a 10MB datafile in /var/lib/mysql/.
+# Read the manual for more InnoDB related options. There are many!
+innodb_buffer_pool_size = 40G
+
+# Replication
+server-id = 1
+log-basename = mariadb
+
+log_bin
+expire_logs_days = 5
+max_binlog_size = 100M
+binlog_format = MIXED
+# the following permits to simplify the process of moving a replica to a
+# primary node role by ensuring that replication is not started on primary node
+skip-slave-start
```

Comment:
The replica to primary move would normally been the former primary is offline. MariaDB won't complain if the primary is offline.  (well every 60 seconds there will be a error in the logs as replication tries to start).

It can be advantageous for the replication to be there on promotion as it ensure that all the replication changes from primary are there. In this case make sure a) the former primary is receiving no updates (starting in --read-only is good), and b) the new master isn't receiving updates until replication is caught up.

So in short I think skip-slave-start might be solving a non-problem, or causing a problem in non-moving scenarios. There may be good cases but without a detailed migration step by step its hard to tell.

Reply:
The `skip-slave-start` option isn't useful only for the secondary node?
Because this configuration file only applies to the Primary node.

I don't know much of how replication was configured. 
Maybe @fauust can help here and give some advice on how we will resume replication after we migrate.

Let me know if I understand correctly your second paragraph: 
 -> on the new bbm host, we start mariadb first, in read-only
 -> we resume replication
 -> if this is ok, we can start bb services

But what if we need to rollback all? 
In case of a rollback we will use the database state of hz-bbm1. So replica will be in a "future" state.
So isn't in our best interest to resume replication only after we validate our services on the new host?

Reply:
resuming replication would be `stop replica; change master to .. (if host changed); start replica`.

yes you could skip replica migration until service validation on the new host.